Agents/FenlandTrajectoryAgent/README.md

```diff
@@ -95,15 +96,42 @@ Importantly, the preprocessing step must be completed before moving on to the in
 
 In addition to the core functionality, this agent offers a route task called layer_generator that creates GeoServer layers from PostGIS tables for visualisation. Given a table name (formed by UUID) and latitude/longitude column names, the agent deploys the [SQL commands for virtual-table generation], creates vector objects, and communicates with the GeoServer REST API to publish the layer. Please note that this functionality is specialized and typically requires manual examination of SQL commands and layer metadata (such as EPSG) to ensure compatibility with specific requirements. This functionality will be further updated and refined after the AI for Public Health (Fenland) project's visualisation is switched to [TWA-VF] 5.4.0.
 
-## Example HTTP Requests
-Example HTTP requests for preprocessing and instantiating data are available in detailed HTTP files. You can access these examples at the [preprocess] file, the [instantiate] file, and the [layer_generator] file. 
+### Example HTTP Requests
+Example requests for preprocessing and instantiating data are available in detailed HTTP files. You can access these examples at the [preprocess] file, the [instantiate] file, and the [layer_generator] file. 
+
+Additionally, services above can be triggered using CURL from a bash terminal. An example CURL command used to load the GPS trajectory files is displayed in [CURL commands folder]. 
+
+## 4.2 Count-Based Exposure Calculation
+
+The agent is configured to calculate exposure by listening to trajectory IRIs and environmental feature IRIs along with a specified exposure radius. 
+The calculation can be triggered via an HTTP request after the agent has started, accessible at http://localhost:{Port}/fenland-trajectory-agent. 
+
+### Functionality Description
+
+The workflow begins by generating a buffer zone around the trajectory geometry, with the buffer extending outward from the trajectory line by a specified exposure radius. Environmental features, such as food retail locations or greenspaces, are then identified and counted based on their intersections with this buffer zone, providing a quantitative measure of exposure. To accommodate different data storage scenarios, the agent provides two routes for exposure calculations:
+
+- **Exposure_count**:  
+This route is used when the environmental data and the Fenland Trajectory Agent are deployed in different stacks. In this case, you need to manually configure the [config.properties] file with the following parameters:
+
+```
+ENV_DATA_ENDPOINT_URL (a SPARQL endpoint)
+TRAJECTORY_DB_HOST
+TRAJECTORY_DB_PASSWORD
+```
```

Comment:
Please remove this list - it is redundant. Simply telling the reader to populate the config file is sufficient. They will see what they need to provide when they look at that file. There is no need to duplicate this information here (and it will only get out of sync when the content of the file is changed!).